StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PrepaneView.swift

```diff
@@ -30,9 +30,13 @@ final class PrepaneView: UIView {
         let paneLayoutView = PaneLayoutView(
             contentView: PaneLayoutView.createContentView(
                 iconView: {
-                    let institutionIconView = InstitutionIconView()
```

Comment:
I think the deleted approach is more elegant, is it possible to let the InstitutionIconView to render dummy view when `setImageUrl(nil)` ?

Reply:
I do believe it renders a dummy (a placeholder image), but what the new code is trying to say is that: "if backend does not return an image, do not render it." The idea is that the backend is telling us that "null means don't render the UI element." 

In terms of not rendering _anything_ when `null` is passed, this is quite tricky (or maybe impossible with `UIKit`). Though `SwiftUI` does give options to handle this nicely (but we use UIKit here).